assets/scss/templates/_guidelines.scss

```diff
@@ -209,3 +209,23 @@ body {
     padding: 5px 0;
   }
 }
+
+
+/* tooltip */
+
+.wrapper-tooltip{
+  flex-wrap: wrap;
+  display: flex;
+  justify-content: space-between;
+  padding: 100px 10px;
+}
+
+.dp-box{
+  @include dp-box($dp-color-silver, 30%, 408px, 250px);
```

Comment:
i dont know if better add a comment to know what means `408px` and `250px` because i dont understand what means

Reply:
In this case it is a mix that we reuse from another component. Perhaps I could make the comments within that mix so that it remains in the documentation and better explain the parameters received.

Reply:
thanks!